src/common/types.ts

```diff
@@ -89,6 +89,7 @@ export type RouteResourceResponseBase<RouteResourceData> = {
   key?: string;
   promise?: RouteResourcePromise<RouteResourceData>;
   expiresAt: RouteResourceTimestamp;
+  accessedAt: RouteResourceTimestamp;
```

Comment:
We also need the Flow types definitions (`.flow` files) for these new keys/options

Reply:
✅